db/migrate/20231106150551_create_reservations.rb

```diff
@@ -2,7 +2,7 @@ class CreateReservations < ActiveRecord::Migration[7.1]
   def change
     create_table :reservations do |t|
       t.references :user, null: false, foreign_key: true
-      t.references :concert, null: false, foreign_key: true
+      t.references :concert, null: false, foreign_key: { on_delete: :cascade}
       t.date :date
```

Comment:
@danifromecuador I think this would delete a `Concert` if a given user wants to delete a `reservation`,  but a Concert may have many reservations, I think would make more sense if the Concert is not deleted if any user cancel or delete their reservation.

Reply:
@danifromecuador My apologies, actually I just figure out this will remove all reservations if the Concert is removed, I'll approve your project now!

Reply:
Thanks, dear reviewer! That's right, if a concert is deleted all its reservations will also be deleted.